npm/src/typings.ts

```diff
@@ -106,6 +106,7 @@ export interface DatabaseOption {
   ttl?: number;
   cleanupLimit?: number;
   encryptionKey?: string;
+  sslRejectUnauthorized?: boolean;
```

Comment:
@niwsa This should be updated in the docs (website)

Reply:
@deepakprabhakara  There is a flag `PGSSLMODE=no-verify` see [link](https://devcenter.heroku.com/articles/connecting-heroku-postgres#connecting-in-node-js). If that's working as expected (need to test) then we can remove the custom handling here for SQL.

Reply:
**PGSSLMODE=no-verify** LGTM. Removed the custom handling.